tests/cli.rs

```diff
@@ -1191,6 +1195,41 @@ fn cert_without_key() {
         .stderr(predicates::str::is_empty());
 }
 
+#[cfg(feature = "online-tests")]
+#[test]
+fn use_ipv4() {
+    get_command()
+        .args(&["https://v6.ipv6-test.com/api/myip.php", "--body", "--ipv4"])
+        .assert()
+        .failure();
+
+    get_command()
+        .args(&[
+            "https://v4v6.ipv6-test.com/api/myip.php",
+            "--body",
+            "--ipv4",
+        ])
+        .assert()
+        .stdout(function(|output: &str| {
+            IpAddr::from_str(output.trim()).unwrap().is_ipv4()
+        }))
+        .stderr(predicates::str::is_empty());
+}
+
+#[cfg(feature = "online-tests")]
+#[test]
+fn use_ipv6() {
```

Comment:
Since not everyone will be to use ipv6 locally, I am wondering if we should gate this behind a feature flag. Maybe `ipv6-tests`?